_insights/labtips.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Practical tips on running a lab
+subtitle: Reflections from the lab team about key areas to think about when developing and running a lab
+permalink: /insights/lab-tips/
+excerpt: Reflections from the lab team about key areas to think about when developing and running a lab
+image: /assets/img/insight/insight-bg.png
+image_accessibility: incon for offering advice
+external_url: "[Project repo](url)"
```

Comment:
Placeholder link 👀

Reply:
There are quite a few of these actually - do they have a function other than as a placeholder?